### PR TITLE
Do not classify IncorrectAccountSequence as network issue

### PR DESCRIPTION
**Pull request by lvn-hasky-dragon**

### CODE REVIEW

**Comment by lvn-rusty-dragon** [comment]
I think we should keep this as a PR until we get some real world data of how this changes works out, agreed?

File: packages/cosmos/src/error.rs [hunk 1]
```diff
@@ -446,7 +446,7 @@ impl QueryErrorDetails {
                     // number.
                     CosmosSdkError::TxInMempool => NetworkIssue,
                     // Similar for account sequence errors
-                    CosmosSdkError::IncorrectAccountSequence => NetworkIssue,
```
Comment (lvn-rusty-dragon) [inline packages/cosmos/src/error.rs:449]: Comment above would need to be modified.

Comment (lvn-hasky-dragon) [reply packages/cosmos/src/error.rs:449]: Thanks, fixed!

**Comment by lvn-hasky-dragon** [comment]
Yeah, agree.

**Review by lvn-rusty-dragon — APPROVED** [review]

**Pull request merged**